mars/services/task/execution/core.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import List, Dict
+from ....resource import Resource, build_band_resources
+
+
+class _CommonPrivateConfigMixin:
```

Comment:
A little strange for me as there is no reason to place a separate Mixin for all backends.

Reply:
Fixed.